public/index.html

```diff
@@ -25,7 +25,7 @@
             <img alt="RChain Community" src="https://avatars1.githubusercontent.com/u/43198800?s=88&v=4">
           </a>
         </div>
-        <div class="progress" style="width: 50%">
+        <div class="progress" style="width: 20%">
```

Comment:
Width again? Looks like you didn't start this branch from master after your earlier work was merged.